app/models/name/format.rb

```diff
@@ -8,6 +8,16 @@ def self.included(base)
   end
 
   ##### Display of names ######################################################
+  def user_display_name(user)
+    str = self[:display_name]
+    if user &&
+       user.hide_authors == "above_species" &&
+       Name.ranks_above_species.include?(rank)
+      str = str.sub(/^(\**__.*__\**).*/, '\\1')
+    end
+    str
+  end
+
   def display_name
```

Comment:
Is this method right? No response needed
```ruby
  def display_name
    str = self[:display_name]
    if User.current &&
       User.current.hide_authors == "above_species" &&
       Name.ranks_above_species.include?(rank)
      str = str.sub(/^(\**__.*__\**).*/, '\\1')
    end
    str
  end
```

Reply:
It looks reasonable to me.  What are you concerned about?